core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -4277,6 +4277,76 @@ private void checkNullOperand(SqlTester tester, String op) {
     tester.checkNull("ASCII(cast(null as varchar(1)))");
   }
 
+  @Test public void testToBase64() {
+    final SqlTester tester1 = tester(SqlLibrary.MYSQL);
```

Comment:
can you pls add test for empty / single char string(s) ?

Reply:
I have addressed. Thanks ^^.